### PR TITLE
Add formatted output

### PR DESCRIPTION
**Pull request by knightdave**
https://github.com/insightsengineering/r.pkg.template/pull/14#issuecomment-974037144

https://github.com/insightsengineering/automation-tasks/issues/151

### CODE REVIEW

**Review by tomszosz — APPROVED** [review]
lgtm

File: action.yaml [hunk 1]
```diff
@@ -188,10 +188,24 @@ runs:
...
+          echo '```'  >> bioccheck_result_formatted.txt
+          grep -A 3 "^Summary:$" bioccheck_result.txt | grep -v Summary >> bioccheck_result_formatted.txt
+          echo -e '```\n' >> bioccheck_result_formatted.txt
+          grep -v "BiocCheck FAILED" bioccheck_result.txt | tail -3 >> bioccheck_result_formatted.txt
+        shell: bash
+
       - uses: marocchino/sticky-pull-request-comment@v2
         with:
           header: BiocCheck
-          path: bioccheck_result.txt
```
Comment (arkadiuszbeer) [inline action.yaml:194]: Why no origin " bioccheck_result.txt" in artifact?

**Review by arkadiuszbeer — APPROVED** [review]
LGTM

File: action.yaml (hunk 1 above)
Comment (knightdave) [reply action.yaml:194]: cause we want to have this formatted (see https://github.com/insightsengineering/r.pkg.template/pull/14#issuecomment-974037144)

**Pull request merged**